### DISCUSSION TITLE
Hey everyone I am facing these issue -

### DISCUSSION THREAD

**Discussion by Kushvanshi-Shubham** [git-lfs/git-lfs#5514] [Q&A]
$ git push origin main
Enumerating objects: 11, done.
Counting objects: 100% (11/11), done.
Delta compression using up to 12 threads
Compressing objects: 100% (10/10), done.
Writing objects: 100% (10/10), 206.41 MiB | 28.83 MiB/s, done.
Total 10 (delta 4), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (4/4), completed with 1 local object.
remote: error: Trace: b36b01354e1220e512f5335689a74de2a09587989479e05207bf1e763dd2c946
remote: error: See https://gh.io/lfs for more information.
remote: error: File Marketplace.zip is 223.10 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.com/Kushvanshi-Shubham/Blockchain.git
 ! [remote rejected] main -> main (pre-receive hook declined)
error: failed to push some refs to 'https://github.com/Kushvanshi-Shubham/Blockchain.git'

**Comment by bk2204** [comment]
Hey,

What to do in this case [is outlined in the FAQ](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-faq.adoc).  If you're still having problems after that, let us know and provide more details of what you've done, and we can provide more assistance.
